src/plugins/discover/public/application/view_components/utils/use_search.ts

```diff
@@ -8,6 +8,7 @@ import { BehaviorSubject, Subject, merge } from 'rxjs';
 import { debounceTime } from 'rxjs/operators';
 import { i18n } from '@osd/i18n';
 import { useEffect } from 'react';
+import { cloneDeep } from 'lodash';
```

Comment:
probably will be apart of a future "remove lodash" epic right

similar issue: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4349

Reply:
yep, this is how the legacy app did it and i didnt want to refactor this right now